templates/index.html

```diff
@@ -126,7 +126,7 @@
                 <div class="panel-body">
                     <div class="tab-content">
                         <div class="tab-pane fade in active home justify" id="home">
-                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}">
+                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}" width="362" height="362">
```

Comment:
width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard